Riot/Modules/Room/Views/InputToolbar/RoomInputToolbarTextView.swift

```diff
@@ -16,8 +16,14 @@
 
 import GrowingTextView
 
+@objc protocol RoomInputToolbarTextViewDelegate: AnyObject {
+    func textView(_ textView: GrowingTextView, didReceivePasteForMediaFromSender sender: Any?)
```

Comment:
Should the textView be a `RoomInputToolbarTextView` type instead?

Reply:
Oh yep, good point!